### ISSUE TITLE
Hachette, Harper, RH updates

### ISSUE THREAD

**Issue by emmbeezee** [almossawi/big-five-publishers#16]
Hi! This is such a fabulous resource - thank you!! Not sure if you're still updating (hoping so!), and if so, wondering if it's possible to update Hachette, adding Workman/Algonquin. And Harper now has HarperAlley. Also, RH/Schwartz & Wade is no longer an imprint - it's now RH Studio and Anne Schwartz Books. Thank you again!!

**Comment by almossawi** [comment]
Hi there, Absolutely!

1. Added Workman and its imprints to Hachette
2. Added HarperAlley to HarperCollins
3. Removed Schwartz & Wade from PRH

Thanks for sharing those updates. I've made the changes and added you to the credits at the bottom (let me know if you'd like me to use a different name). You might want to shift+refresh to see the updated graphics :)

<img width="1082" alt="Screenshot 2023-03-22 at 5 20 06 PM" src="https://user-images.githubusercontent.com/980049/227067979-dbbb4a33-6b05-4a07-b42c-c01bfe6f2479.png">
<img width="1064" alt="Screenshot 2023-03-22 at 5 15 56 PM" src="https://user-images.githubusercontent.com/980049/227067985-f27b3707-9a38-4a6c-8890-939c5badc0c1.png">
<img width="1073" alt="Screenshot 2023-03-22 at 5 11 54 PM" src="https://user-images.githubusercontent.com/980049/227067990-61d93d9a-33f4-4838-ad30-8b07a4654359.png">
